### ISSUE TITLE
Build newer cvmfs2 release for PowerPC

### ISSUE THREAD

**Issue by mrodozov** [cvmfs/cvmfs#2564]
Dears,

we have installed 2.3.3 version on a development powerpc machine, which is working, but often fails (every 3 days on average) and the client needs restart or the machine - reboot, when the client remount doesn't help. 
I tried building it myself on that machine:

```
lscpu
Architecture:          ppc64le
Byte Order:            Little Endian
CPU(s):                128
On-line CPU(s) list:   0-127
Thread(s) per core:    8
Core(s) per socket:    8
Socket(s):             2
NUMA node(s):          2
Model:                 1.0 (pvr 004c 0100)
Model name:            POWER8NVL (raw), altivec supported
```

but I didn't know many details about the requirements like what version of fuse goes for what version of cvmfs, how to deal with the missing types from capability.h or how you dealt with it when you built 2.3.3, how to set cmake vars such that not only to configure but to build without complains that something is not found (although it was set with cmake).

In short - could you build new version for powerpc ? Newer client versions has proven to work better, we had the same issue on arm and installing 2.6 fixed it.

If not - could you help me do it if I explain the details (where I am and what is missing) ? 
Thank you in advance,
Cheers,
Mircho

**Comment by siscia** [comment]
Hi Mircho,

I am not aware of any readily available powerpc machine that we can use for building at the moment.

However the build is not complex, and we can definitely help with that.

In short it is something like:

```
git clone https://github.com/cvmfs/cvmfs
cd cvmfs
mkdir build
cd build
cmake ..
make -j(n-2)
```

There are some dependencies that need to be installed but cmake should point out what is missing.

I shoot you a message over mattermost. Feel free to reach for help over there as well.

Cheers,
Simone

**Comment by mrodozov** [comment]
Hi again, I just built all targets, had to install libcap-devel which brings sys/capability.h 
the config is: 
`cmake .. -DUUID_INCLUDE_DIR=/usr/include/linux/ 
 -DUUID_LIBRARY=/lib64/libuuid.so.1 
 -DFUSE_INCLUDE_DIR=/scratch/mrodozov/fuse3_rpm/usr/include/ 
 -DCAP_LIBRARY=/lib64/libcap.so.2 
 -DCAP_INCLUDE_DIR=/usr/src/kernels/3.10.0-957.21.3.el7.ppc64le/include/uapi/linux`
One needs also fuse3-devel and fuse3-libs which in my case are locally installed.
Then change the generated -DCVMFS_USE_LIBFUSE=2 to -DCVMFS_USE_LIBFUSE=3
in flags.make generated in the build dir, otherwise it searches the fuse2 headers and fails.
Thats for future reference.
